gnovm/pkg/gnoimports/package.go

```diff
@@ -0,0 +1,156 @@
+package gnoimports
```

Comment:
Since we'll never have the `gnoimports` binary and only have a single full-featured `gno fmt` that includes `go fmt`, `go imports`, and later custom flavors, let's rename this package `gnofmt`.